ledger/apply/keyreg_test.go

```diff
@@ -136,20 +136,20 @@ func TestKeyregApply(t *testing.T) {
 
 	// Going from offline to online should be okay
 	mockBal.addrs[src] = basics.AccountData{Status: basics.Offline}
-	err = Keyreg(tx.KeyregTxnFields, tx.Header, mockBal, transactions.SpecialAddresses{FeeSink: feeSink}, nil, basics.Round(0))
+	err = Keyreg(tx.KeyregTxnFields, tx.Header, mockBal, transactions.SpecialAddresses{}, nil, basics.Round(0))
```

Comment:
Why are we removing the FeeSink from special addresses?

Reply:
This happened while I was pursuing another way to do this PR, but I think the cleanup is useful anyway.  These tests do not care what the `FeeSink` is set to, so I think it's nicer not to set it at all.  It streamlines what the reader pays attention to.